cluster/images/controller-manager/Dockerfile

```diff
@@ -33,7 +33,7 @@ ARG DISTROLESS_IMAGE=gcr.io/distroless/static@sha256:9b60270ec0991bc4f14bda475e8
 FROM ${GOLANG_IMAGE} as builder
 
 # This build arg is the version to embed in the CPI binary
-ARG VERSION=unknown
+ARG VERSION=1.22.3
```

Comment:
for local development, if you run `docker build` command without version info, the log is still not correct. Probably we can leave the fixed value and update it when we have a new release version